packages/auth/src/constants.ts

```diff
@@ -8,46 +8,6 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-export const REDIS_HOST = process.env.REDIS_HOST || 'localhost'
-export const AUTH_HOST = process.env.AUTH_HOST || '0.0.0.0'
-export const AUTH_PORT = process.env.AUTH_PORT || 4040
-export const USER_MANAGEMENT_URL =
-  process.env.USER_MANAGEMENT_URL || 'http://localhost:3030/'
-export const METRICS_URL = process.env.METRICS_URL || 'http://localhost:1050'
-export const NOTIFICATION_SERVICE_URL =
-  process.env.NOTIFICATION_SERVICE_URL || 'http://localhost:2020/'
-export const HOSTNAME = process.env.DOMAIN || '*'
-export const COUNTRY_CONFIG_URL =
-  process.env.COUNTRY_CONFIG_URL || 'http://localhost:3040/'
-export const LOGIN_URL = process.env.LOGIN_URL || 'http://localhost:3020/'
-export const CLIENT_APP_URL =
-  process.env.CLIENT_APP_URL || 'http://localhost:3000/'
-
-export const CERT_PRIVATE_KEY_PATH =
-  (process.env.CERT_PRIVATE_KEY_PATH as string) ||
-  '../../.secrets/private-key.pem'
-export const CERT_PUBLIC_KEY_PATH =
-  (process.env.CERT_PUBLIC_KEY_PATH as string) ||
-  '../../.secrets/public-key.pem'
-export const SENTRY_DSN = process.env.SENTRY_DSN
-
-export const PRODUCTION = process.env.NODE_ENV === 'production'
-export const QA_ENV = process.env.QA_ENV || false
-
-export const CONFIG_TOKEN_EXPIRY_SECONDS = process.env
-  .CONFIG_TOKEN_EXPIRY_SECONDS
-  ? parseInt(process.env.CONFIG_TOKEN_EXPIRY_SECONDS, 10)
-  : 604800 // 1 week
-
-export const CONFIG_SMS_CODE_EXPIRY_SECONDS = process.env
-  .CONFIG_SMS_CODE_EXPIRY_SECONDS
-  ? parseInt(process.env.CONFIG_SMS_CODE_EXPIRY_SECONDS, 10)
-  : 600
-
-export const CONFIG_SYSTEM_TOKEN_EXPIRY_SECONDS = process.env
-  .CONFIG_SYSTEM_TOKEN_EXPIRY_SECONDS
-  ? parseInt(process.env.CONFIG_SYSTEM_TOKEN_EXPIRY_SECONDS, 10)
-  : 600
 
 export const WEB_USER_JWT_AUDIENCES = [
```

Comment:
Should we aim to move these leftover constants closer to where they are needed? These are related to creating the JWT so they should be defined [here](https://github.com/opencrvs/opencrvs-core/blob/62df49f96575d861bb199e48334282a65a447e87/packages/auth/src/features/authenticate/service.ts)

Reply:
@naftis yes I think so, if the are not configurable

Reply:
I'll leave them here for now as the `authenticate/service.ts` also isn't the best place, as many of the other handlers/services also refer to `authenticate/service.ts`, we should rather have an `authenticate.ts` which is referred by the services/handlers. I remember refactoring this in the MOSIP pre-work so lets touch it when I get back to that work.